bats/fb-content-katello.bats

```diff
@@ -284,3 +284,191 @@ setup() {
   CONTAINER_PULL_LABEL=`echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]'`
   podman pull "${HOSTNAME}/${CONTAINER_PULL_LABEL}"
 }
+
+@test "create and sync modules-rpms repo" {
+  hammer repository create --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY_2}" \
+    --url https://partha.fedorapeople.org/test-repos/separated/modules-rpms/ | grep -q "Repository created"
+  hammer repository synchronize --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --name="${YUM_REPOSITORY_2}"
+}
+
+@test "create and sync rpm-deps repo" {
+  hammer repository create --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY_3}" \
+    --url https://partha.fedorapeople.org/test-repos/separated/rpm-deps/ | grep -q "Repository created"
```

Comment:
would it be easy enough to change to use https://fixtures.pulpproject.org/  ?   i'd rather do that than a dev's fedorapeople page.  If they don't have the needed content to test this, than its fine

Reply:
And if they don't, are they interested in hosting them?

Reply:
That's an excellent site by them. If they don't have what's needed, ask them :) That site seems the perfect spot for all incantations of content repos needed.

Reply:
I'll ask them if they have a repo pair they use for testing dep solving.  If not, maybe they can host this one.

Reply:
Put in the request here: https://pulp.plan.io/issues/7333

Reply:
lets go ahead and merge this as is, it seems adding repos to the pulp fixtures is non-trivial

Reply:
Might be nice to add the link to that issue in a TODO inside the file somewhere.

Reply:
Added the TODO.